config/routes.rb

```diff
@@ -57,6 +57,7 @@
             post :clone
             post :attach_file, on: :collection
           end
+          resources :macros
```

Comment:
let's also explicitly define the methods we use in controller using `only:` so that we don't accidentally leave any endpoints open